src/hotspot/share/opto/library_call.cpp

```diff
@@ -29,7 +29,9 @@
 #include "compiler/compileBroker.hpp"
 #include "compiler/compileLog.hpp"
 #include "gc/shared/barrierSet.hpp"
+#if INCLUDE_JFR
```

Comment:
It is custom to put the conditional includes at the end, not inlined (please see other files for conditional includes). Thanks.

Reply:
fixed